dpnp/linalg/dpnp_utils_linalg.py

```diff
@@ -260,99 +260,78 @@ def _batched_solve(a, b, exec_q, res_usm_type, res_type):
 
     """
 
-    a_usm_arr = dpnp.get_usm_ndarray(a)
-    b_usm_arr = dpnp.get_usm_ndarray(b)
-
-    b_order = "C" if b.flags.c_contiguous else "F"
     a_shape = a.shape
     b_shape = b.shape
 
-    is_cpu_device = exec_q.sycl_device.has_aspect_cpu
-    reshape = False
-    orig_shape_b = b_shape
-    if a.ndim > 3:
-        # get 3d input arrays by reshape
-        if a.ndim == b.ndim:
-            b = dpnp.reshape(b, (-1, b_shape[-2], b_shape[-1]))
-        else:
-            b = dpnp.reshape(b, (-1, b_shape[-1]))
+    # gesv_batch expects `a` to be a 3D array and
+    # `b` to be either a 2D or 3D array.
+    if a.ndim == b.ndim:
+        b = b.reshape(-1, b_shape[-2], b_shape[-1])
+    else:
+        b = b.reshape(-1, b_shape[-1])
 
-        a = dpnp.reshape(a, (-1, a_shape[-2], a_shape[-1]))
+    a = a.reshape(-1, a_shape[-2], a_shape[-1])
```

Comment:
The same question here if it is secured that `a` is always dpnp array? Is is covered by any test?